packages/app/src/Const.ts

```diff
@@ -168,4 +168,4 @@ export const MagnetRegex = /(magnet:[\S]+)/i;
 /**
  * Wavlake embed regex
  */
-export const WavlakeRegex = /player\.wavlake\.com\/(track)\/([.a-zA-Z0-9-]+)/;
+export const WavlakeRegex = /player\.wavlake\.com\/([.a-zA-Z0-9-/]+)/i;
```

Comment:
What do you think about making this regex more specific? This is pretty general and even matches on the RSS feed links e.g. https://player.wavlake.com/feed/e4c5ff61-ef60-40e9-a289-82b1027824ae. If anyone ever shared their RSS feed link, it would display an iframe with a 404 instead of their RSS feed link.